upgrade to 1.0.3

saraedum opened the pull request.

saraedum committed:
upgrade to 1.0.3

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

saraedum commented:
@conda-forge-admin, please rerender.

Commit:
MNT: Re-rendered with conda-build 3.21.4, conda-smithy 3.14.3, and conda-forge-pinning 2021.11.07.12.06.04

saraedum committed:
upgrade sha256 sum

saraedum committed:
do not test on emulators

it does not seem to work. Probably because the package actually does not
work but that seems to be a problem with cppyy and not with e-antic.

saraedum commented:
@conda-forge-admin, please rerender.

Commit:
MNT: Re-rendered with conda-build 3.21.4, conda-smithy 3.14.3, and conda-forge-pinning 2021.11.07.12.06.04
